### PR TITLE
Jtj 55.redesign

### PR DESCRIPTION
**Pull request by tgi-wtag**
This is my PR regarding a redesign of the start page, which is now a dashboard displaying the journal entries from today, the goals and a todolist.

https://github.com/tgi-wtag/Journey-tg/assets/141820846/0260495c-2399-4fc7-8313-f80e6d896a76

thank you in advance for giving a quick review.:sunflower:

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (evk-wtag) [inline app/assets/javascripts/application.js:1]: Remove extra line

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,12 +1,54 @@
...
+      <% today_journal_entries = current_user.journal_entries.where(date: Date.today) %>
+      <div class="content-container">
+        <% today_journal_entries.each do |entry| %>
+          <div class="content-card">
+            <h3><%= entry.title %></h3>
+            <p><%= entry.content %></p>
+          </div>
+        <% end %>
+      </div>
+      <h2><%= t('goals_today') %></h2>
```
Comment (evk-wtag) [inline app/views/pages/home.html.erb:15]: Let's add the keys to the translation files

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -1,12 +1,54 @@
...
+          <div class="content-card">
+            <h3><%= goal.description %></h3>
+            <p><%= goal.status %></p>
+          </div>
+        <% end %>
+      </div>
+    <% else %>
+      <div class="content-container">
+        <%= link_to new_user_path, class: "content-card" do %>
+          <%= t('create_user') %>
```
Comment (evk-wtag) [inline app/views/pages/home.html.erb:28]: Ditto

**Review by evk-wtag — APPROVED** [review]
Let's add some suitable texts for the Journal entries using FactoryBot or manually

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (tgi-wtag) [reply app/assets/javascripts/application.js:1]: thanks for pointing out

File: app/views/pages/home.html.erb (hunk 2 above)
Comment (tgi-wtag) [reply app/views/pages/home.html.erb:15]: thank you, i did

File: app/views/pages/home.html.erb (hunk 3 above)
Comment (tgi-wtag) [reply app/views/pages/home.html.erb:28]: thank you

**Review by chm-wtag — APPROVED** [review]

**Comment by chm-wtag** [comment]
resolve the merge conflicts please

**Pull request merged**